pandas/io/tests/parser/common.py

```diff
@@ -439,6 +439,13 @@ def test_get_chunk_passed_chunksize(self):
         piece = result.get_chunk()
         self.assertEqual(len(piece), 2)
 
+    def test_read_chunksize_generated_index(self):
+        # GH 12185
```

Comment:
Do you mean the fix or the test? Anywhere, both the fix and the test apply to both parsers.